packages/template-retail-react-app/app/hooks/use-auth-modal.js

```diff
@@ -20,7 +20,15 @@ import {
     useDisclosure,
     useToast
 } from '@chakra-ui/react'
-import useCustomer from '../commerce-api/hooks/useCustomer'
+import {
+    ShopperLoginHelpers,
+    useShopperLoginHelper,
+    useCustomer,
+    useCustomerId,
+    useCustomerType,
+    useShopperCustomersMutation,
+    ShopperCustomersMutations
+} from 'commerce-sdk-react-preview'
```

Comment:
NIT: My OCD is telling me that `ShopperLoginHelpers` should be next to `ShopperCustomersMutations` 🤪

Reply:
Side not, I wonder if the eslint sort-imports plugin would help use clean up our imports a little.